lua/cmake/init.lua

```diff
@@ -45,7 +45,7 @@ function cmake.build(args)
   end
 
   args = args or {}
-  vim.list_extend(args, { '--build', project_config:get_build_dir().filename, '--target', project_config.json.current_target, unpack(config.build_args) })
+  args = vim.list_extend({ '--build', project_config:get_build_dir().filename, '--target', project_config.json.current_target, unpack(config.build_args) }, args)
```

Comment:
Could you remove previous line and write it like this?
```suggestion
  args = vim.list_extend({ '--build', project_config:get_build_dir().filename, '--target', project_config.json.current_target, unpack(config.build_args) }, args or {})
```